packages/swirl-components/src/components/swirl-table/swirl-table.spec.tsx

```diff
@@ -2,7 +2,7 @@ import { newSpecPage } from "@stencil/core/testing";
 
 import { SwirlTable } from "./swirl-table";
 
-(global as any).MutationObserver = class {
+(global as any).IntersectionObserver = class {
```

Comment:
**note:**  I removed the mocking of the MutationObserver, since it is not used in the table component. And replaced it with the mocking of the newly added IntersectionObserver.